File: PosBiptek/app/src/main/java/com/biptek/posbiptek/model/Restock.java
package com.biptek.posbiptek.model;

public class Restock {
    private long kode_restock;
    private long kode_supplier_restock;
    private String username_pegawai_restock;
    private String tanggal_transaksi_restock;
    private String tanggal_jatuh_tempo;
    private String bukti_transaksi_restock;

    public Restock(long kode_restock, long kode_supplier_restock, String username_pegawai_restock, String tanggal_transaksi_restock, String tanggal_jatuh_tempo, String bukti_transaksi_restock) {
        this.kode_restock = kode_restock;
        this.kode_supplier_restock = kode_supplier_restock;
        this.username_pegawai_restock = username_pegawai_restock;
        this.tanggal_transaksi_restock = tanggal_transaksi_restock;
        this.tanggal_jatuh_tempo = tanggal_jatuh_tempo;
        this.bukti_transaksi_restock = bukti_transaksi_restock;
    }

    public Restock() {

    }

    public long getKode_restock() {
        return kode_restock;
    }

    public void setKode_restock(long kode_restock) {
        this.kode_restock = kode_restock;
    }

    public long getKode_supplier_restock() {
        return kode_supplier_restock;
    }

    public void setKode_supplier_restock(long kode_supplier_restock) {
        this.kode_supplier_restock = kode_supplier_restock;
    }

    public String getUsername_pegawai_restock() {
        return username_pegawai_restock;
    }

    public void setUsername_pegawai_restock(String username_pegawai_restock) {
        this.username_pegawai_restock = username_pegawai_restock;
    }

    public String getTanggal_transaksi_restock() {
        return tanggal_transaksi_restock;
    }

    public void setTanggal_transaksi_restock(String tanggal_transaksi_restock) {
        this.tanggal_transaksi_restock = tanggal_transaksi_restock;
    }

    public String getTanggal_jatuh_tempo() {
        return tanggal_jatuh_tempo;
    }

    public void setTanggal_jatuh_tempo(String tanggal_jatuh_tempo) {
        this.tanggal_jatuh_tempo = tanggal_jatuh_tempo;
    }

    public String getBukti_transaksi_restock() {
        return bukti_transaksi_restock;
    }

    public void setBukti_transaksi_restock(String bukti_transaksi_restock) {
        this.bukti_transaksi_restock = bukti_transaksi_restock;
    }

    @Override
    public String toString() {
        return "Restock{" +
                "kode_restock=" + kode_restock +
                ", kode_supplier_restock=" + kode_supplier_restock +
                ", username_pegawai_restock='" + username_pegawai_restock + '\'' +
                ", tanggal_transaksi_restock='" + tanggal_transaksi_restock + '\'' +
                ", tanggal_jatuh_tempo='" + tanggal_jatuh_tempo + '\'' +
                ", bukti_transaksi_restock='" + bukti_transaksi_restock + '\'' +
                '}';
    }
}

File: Server/advanced/backend/models/Restock.php
<?php

namespace backend\models;

use Yii;

/**
 * This is the model class for table "restock".
 *
 * @property int $kode_restock
 * @property int $kode_supplier_restock
 * @property string $username_pegawai_restock
 * @property string $tanggal_transaksi_restock
 * @property string $tanggal_jatuh_tempo
 * @property string $bukti_transaksi_restock
 *
 * @property ListProdukRestock[] $listProdukRestocks
 * @property Supplier $kodeSupplierRestock
 * @property Pegawai $usernamePegawaiRestock
 */
class Restock extends \yii\db\ActiveRecord
{
    /**
     * {@inheritdoc}
     */
    public static function tableName()
    {
        return 'restock';
    }

    /**
     * {@inheritdoc}
     */
    public function rules()
    {
        return [
            [['kode_supplier_restock', 'username_pegawai_restock', 'tanggal_transaksi_restock', 'tanggal_jatuh_tempo'], 'required'],
            [['kode_supplier_restock'], 'integer'],
            [['tanggal_transaksi_restock', 'tanggal_jatuh_tempo'], 'safe'],
            [['username_pegawai_restock'], 'string', 'max' => 25],
            [['bukti_transaksi_restock'], 'string', 'max' => 50],
            [['kode_supplier_restock'], 'exist', 'skipOnError' => true, 'targetClass' => Supplier::className(), 'targetAttribute' => ['kode_supplier_restock' => 'kode_supplier']],
            [['username_pegawai_restock'], 'exist', 'skipOnError' => true, 'targetClass' => Pegawai::className(), 'targetAttribute' => ['username_pegawai_restock' => 'username_pegawai']],
        ];
    }

    /**
     * {@inheritdoc}
     */
    public function attributeLabels()
    {
        return [
            'kode_restock' => 'Kode Restock',
            'kode_supplier_restock' => 'Kode Supplier Restock',
            'username_pegawai_restock' => 'Username Pegawai Restock',
            'tanggal_transaksi_restock' => 'Tanggal Transaksi Restock',
            'tanggal_jatuh_tempo' => 'Tanggal Jatuh Tempo',
            'bukti_transaksi_restock' => 'Bukti Transaksi Restock',
        ];
    }

    /**
     * @return \yii\db\ActiveQuery
     */
    public function getListProdukRestocks()
    {
        return $this->hasMany(ListProdukRestock::className(), ['kode_restock_list' => 'kode_restock']);
    }

    /**
     * @return \yii\db\ActiveQuery
     */
    public function getKodeSupplierRestock()
    {
        return $this->hasOne(Supplier::className(), ['kode_supplier' => 'kode_supplier_restock']);
    }

    /**
     * @return \yii\db\ActiveQuery
     */
    public function getUsernamePegawaiRestock()
    {
        return $this->hasOne(Pegawai::className(), ['username_pegawai' => 'username_pegawai_restock']);
    }
}
